core/plugins/org.polarsys.capella.core.ui.properties/src/org/polarsys/capella/core/ui/properties/providers/CapellaTransfertViewerLabelProvider.java

```diff
@@ -39,25 +46,32 @@
 import org.polarsys.capella.core.model.handler.helpers.CapellaProjectHelper.TriStateBoolean;
 import org.polarsys.capella.core.model.helpers.ComponentExchangeExt;
 import org.polarsys.capella.core.ui.properties.CapellaUIPropertiesPlugin;
+import org.polarsys.capella.core.ui.resources.CapellaUIResourcesPlugin;
 
 public class CapellaTransfertViewerLabelProvider extends DataLabelProvider {
 
   private static String PATTERN1 = " [{0} -> {1}]{2}"; //$NON-NLS-1$
   private static String UNAMED = "<unnamed>"; //$NON-NLS-1$
-  
+
   private boolean disableLabelComputation = CapellaUIPropertiesPlugin.getDefault().isDisableLabelComputation();
 
   /**
    * {@inheritDoc}
    */
   @Override
   public Image getImage(Object object) {
-    if (object instanceof DView) { //Sirius-2822
+    if (object instanceof DView) { // Sirius-2822
       Viewpoint viewpoint = ((DView) object).getViewpoint();
       if (viewpoint != null) {
         return getImage(viewpoint);
       }
     }
+    if (object instanceof DAnnotation) {
```

Comment:
Could you check that this should apply to only DAnnotations that represent title blocks. Your code applies to all DAnnotation which is not what we want.

Reply:
Done.